lib/appoptics.js

```diff
@@ -290,6 +289,7 @@ var flushStats = function appopticsFlush(ts, metrics) {
     // Valid format for parsing tags out: global-prefix.name#tag1=value,tag2=value
     // NOTE: Name can include the source
     var vals = measureName.split('#');
+    Object.assign(measure.tags, tags);
```

Comment:
wanted to use spread operator here but travis appears to check node version 4/6 (not available) and latest stable.